source/en/calibration_freshwater/SDR_script_example_manual_repeat.py

```diff
@@ -0,0 +1,55 @@
+    'drainage_path': '',
+    'erodibility_path': 'G:\\Shared drives\\GEF_GreenFin\\Pilot_Countries\\Philippines\\DENR_Stanford_sharing\\SDR_Luzon_example_inputs\\soil_erodibility_ISRIC_Luzon.tif',
+    'erosivity_path': 'G:\\Shared drives\\GEF_GreenFin\\Pilot_Countries\\Philippines\\DENR_Stanford_sharing\\SDR_Luzon_example_inputs\\rainfall_erosivity_WorldClim_Luzon.tif',
+    'ic_0_param': 0.5,
+    'k_param': 2,
+    'l_max': 122,
+    'lulc_path': 'G:\\Shared drives\\GEF_GreenFin\\Pilot_Countries\\Philippines\\DENR_Stanford_sharing\\SDR_Luzon_example_inputs\\LULC_2020_Luzon.tif',
+    'results_suffix': 'Luzon_example',
+    'sdr_max': 0.8,
+    'threshold_flow_accumulation': 1000,
+    'watersheds_path': 'G:\\Shared drives\\GEF_GreenFin\\Pilot_Countries\\Philippines\\DENR_Stanford_sharing\\SDR_Luzon_example_inputs\\Luzon_AOI.shp',
+    'workspace_dir': 'G:\\Shared drives\\GEF_GreenFin\\Pilot_Countries\\Philippines\\DENR_Stanford_sharing\\Calibration\\SDR_results', # Filepaths to each baseline model input
+} # UPDATE these filepaths to reflect the absolute paths on your computer
+
+if __name__ == '__main__':
+    natcap.invest.sdr.sdr.execute(args) # Run InVEST SDR model
+
+args = {
+    'biophysical_table_path': 'G:\\Shared drives\\GEF_GreenFin\\Pilot_Countries\\Philippines\\DENR_Stanford_sharing\\SDR_Luzon_example_inputs\\biophysical_table_SDR_Yang2003_Luzon.csv',
+    'dem_path': 'G:\\Shared drives\\GEF_GreenFin\\Pilot_Countries\\Philippines\\DENR_Stanford_sharing\\SDR_Luzon_example_inputs\\DEM_SRTM_Luzon_90m.tif',
+    'drainage_path': '',
+    'erodibility_path': 'G:\\Shared drives\\GEF_GreenFin\\Pilot_Countries\\Philippines\\DENR_Stanford_sharing\\SDR_Luzon_example_inputs\\soil_erodibility_ISRIC_Luzon.tif',
+    'erosivity_path': 'G:\\Shared drives\\GEF_GreenFin\\Pilot_Countries\\Philippines\\DENR_Stanford_sharing\\SDR_Luzon_example_inputs\\rainfall_erosivity_WorldClim_Luzon.tif',
+    'ic_0_param': 0.5,
+    'k_param': 1.8, # parameter value changed from run above
+    'l_max': 122,
+    'lulc_path': 'G:\\Shared drives\\GEF_GreenFin\\Pilot_Countries\\Philippines\\DENR_Stanford_sharing\\SDR_Luzon_example_inputs\\LULC_2020_Luzon.tif',
+    'results_suffix': 'Luzon_example_kb1_8', # results filenames adjusted to reflect parameter change from baseline value
+    'sdr_max': 0.8,
+    'threshold_flow_accumulation': 1000,
+    'watersheds_path': 'G:\\Shared drives\\GEF_GreenFin\\Pilot_Countries\\Philippines\\DENR_Stanford_sharing\\SDR_Luzon_example_inputs\\Luzon_AOI.shp',
+    'workspace_dir': 'G:\\Shared drives\\GEF_GreenFin\\Pilot_Countries\\Philippines\\DENR_Stanford_sharing\\Calibration\\SDR_results', # Filepaths to each model input for adjusted parameter run
+} # UPDATE these filepaths to reflect the absolute paths on your computer
+
+if __name__ == '__main__':
+    natcap.invest.sdr.sdr.execute(args) # Run InVEST SDR model
```

Comment:
This script is pretty similar to the next one, so I might suggest only including that one and suggesting people save a separate script for each scenario. Or if you think it's useful to run multiple models in one script like this, I would suggest the following: Two separate arguments dictionaries (e.g. `args_scenario1` and `args_scenario2`). And then a single,

```
if __name__ == '__main__':
    natcap.invest.sdr.sdr.execute(args_scenario1)
    natcap.invest.sdr.sdr.execute(args_scenario2)
```

Reply:
Good points, I agree with Dave here, either suggest separate scripts for each scenario or to use different argument dictionaries. I lean towards the latter as I think fewer scripts would be simpler.

Reply:
It is, I was also wondering about whether to keep it, deleted its reference from the chapter.